q2_diversity/plugin_setup.py

```diff
@@ -138,6 +138,7 @@
     function=q2_diversity.beta,
     inputs={'table': FeatureTable[Frequency]},
     parameters={'metric': Str % Choices(beta.non_phylogenetic_metrics()),
+                'pseudocount': Int,
```

Comment:
Should a range be applied to this? Setting a pseudocount of 0 seems silly. As well, would there be a use case for negative pseudocounts? If not, you could do something like this:

```python
from qiime2.plugin import Range

...

'pseudocount': Int % Range(1, None),
```

Reply:
👍 Although in future cases we will want to handle scenarios to have fractional pseudocounts, which is why I'm pushing in a PR into skbio to relax the type checks